<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AI CAM</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="videoTab">
       <attribute name="title">
        <string>Video</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QGridLayout" name="videoStreamLayout"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="imagesTab">
       <attribute name="title">
        <string>Images</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="3" column="0" colspan="2">
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>722</width>
             <height>136</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QListWidget" name="imagesListWidget"/>
        </item>
        <item row="1" column="1">
         <widget class="QListWidget" name="labelsListWidget"/>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QPushButton" name="imageEditButton">
          <property name="text">
           <string>Edit</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="reloadButton">
          <property name="text">
           <string>reload</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="trainTab">
       <attribute name="title">
        <string>Train</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <layout class="QGridLayout" name="neuralNetLayout"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="alarmTab">
       <attribute name="title">
        <string>Alarm</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QGridLayout" name="alarmLayout"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="loggerGridLayout"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>imagesListWidget</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>labelsListWidget</receiver>
   <slot>update(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>264</y>
    </hint>
    <hint type="destinationlabel">
     <x>513</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
